spec/requests/sessions_requests_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "Sessions", type: :request, order: :defined do
+  context "Sign in as an org_admin before signing in as a partner" do
+    before do
+      # sign in as an org_admin
+      sign_in(@organization_admin)
+
+      # sign in as a partner
+      user = User.create!(
+        email: "partner123@example.com",
+        name: "Test Partner",
+        password: "password!"
+      )
+      user.add_role(:partner, create(:organization))
```

Comment:
uh this seems wrong. The `partner` role is for partners, not organizations. The fact that this works makes me feel wonky. :woozy\_face:

Reply:
Hmm, yes -- @Kernapillar should make it point to a partner, not an organization, but we (i.e. , to be realistic, @dorner) should look into why that would even work. Does the role know what sort of resource it is pointing at?

Reply:
No, it does not, and I'm not sure we can enforce it.

Reply:
Okay, I can go back and take another look!

Reply:
I just simplified the spec file, I think I was over complicating with the last version. 

Please let me know if anything else should be added!